### PR TITLE
PHP Unit 6

### PR DESCRIPTION
**Pull request by mcneely**
Updated Tests to run up to 6.5.
Addresses Issue #45

### CODE REVIEW

File: tests/BaseTest.php [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 namespace PHPChunkit\Test;
 
-use PHPUnit_Framework_TestCase;
+require_once __DIR__ . '/bootstrap.php';
+use PHPUnit\Framework\TestCase;
```
Comment (jwage) [inline tests/BaseTest.php]: Can you add a new line between the require_once and the use statement?

File: tests/bootstrap.php [hunk 2]
```diff
@@ -1,3 +1,7 @@
 <?php
 
 require_once dirname(__DIR__) . '/vendor/autoload.php';
+
+if (class_exists('PHPUnit_Framework_TestCase') && !class_exists('PHPUnit\Framework\TestCase')) {
+    class_alias( 'PHPUnit_Framework_TestCase', 'PHPUnit\Framework\TestCase');
+}
```
Comment (jwage) [inline tests/bootstrap.php]: Can you add a new line at the end of this file?

File: tests/BaseTest.php (hunk 1 above)
Comment (mcneely) [reply tests/BaseTest.php]: Done.

File: tests/bootstrap.php (hunk 2 above)
Comment (mcneely) [reply tests/bootstrap.php]: Done.

**Comment by mcneely** [comment]
Version might need to be adjust slightly to deal with the scrutinizer issue, otherwise if the tests are passing then we should be fine.

**Pull request merged**